apps/myaccount/src/extensions/configs/common.ts

```diff
@@ -47,6 +48,13 @@ export const commonConfig: CommonConfig = {
         },
         isConsoleNavigationAllowed(userstore: string): boolean {
             return true;
+        },
+        getProfileCompletion(
+            profileInfo?: BasicProfileInterface,
+            profileSchemas?: ProfileSchema[],
+            isReadOnlyUser?: boolean
+        ): undefined {
+            return undefined;
```

Comment:
This shouldn't return undefined. I think it will be better if we can implement the logic that is unique to identity apps here.